src/main/java/org/openbmp/handler/Headers.java

```diff
@@ -0,0 +1,61 @@
+package org.openbmp.handler;
+
+import org.openbmp.helpers.IpAddr;
+import org.openbmp.processor.ParseLongEmptyAsZero;
+import org.openbmp.processor.ParseNullAsEmpty;
+import org.openbmp.processor.ParseTimestamp;
+import org.supercsv.cellprocessor.ParseLong;
+import org.supercsv.cellprocessor.constraint.NotNull;
+import org.supercsv.cellprocessor.ift.CellProcessor;
+
+/**
+ * Created by ALEX on 12/2/15.
+ */
+public class Headers {
+
+    private String version;
+
+    private String collector_hash_id;
+
+    private long length;
+
+    private long records;
+
+    /**
+     * Handle the headers by parsing it and storing the data in memory.
+     *
+     * @param data
+     */
+    public Headers(String data) {
+        parse(data);
+    }
+
+    /**
+     * Setting the headers' values
+     *
+     * @param data
+     */
+    private void parse(String data) {
```

Comment:
The order may be different.  You need to parse each key:value.  For example, it's allowed to have R before L or L before R, V could come last, etc...